### PR TITLE
Added memberinfo

### PR DESCRIPTION
**Pull request by BHLoneWolf0**

### CODE REVIEW

File: docs/xf8bot/commands/info/memberinfo.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Member Info Command
+permalink: /xf8bot/commands/info/memberinfo/
+---
+
+# Information Command
+
+The `memberinfo` command tells you the information about the members.
```
Comment (xf8b) [inline docs/xf8bot/commands/info/memberinfo.md:9]: what members? be more specific

**Review by xf8b — CHANGES_REQUESTED** [review]

File: docs/xf8bot/commands/info/memberinfo.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Member Info Command
+permalink: /xf8bot/commands/info/memberinfo/
+---
+
+# Information Command
+
+The `memberinfo` command tells you the information about the tells information about the member, such as their roles or when they joined the server.
```
Comment (xf8b) [inline docs/xf8bot/commands/info/memberinfo.md:9]: tells you information about the tells information 🤔

File: docs/xf8bot/commands/info/memberinfo.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Sends the information about the member passed in.   
+Currently contains: 
+
+* Activity
+* Status
+* Roles
+* Avatar URL
+* Joined Discord
+* Joined server
```
Comment (xf8b) [inline docs/xf8bot/commands/info/memberinfo.md:25]: i havent checked but this seems like its missin a few things

**Review by xf8b — CHANGES_REQUESTED** [review]
good changes, just got a couple more requests

File: docs/xf8bot/commands/info/memberinfo.md (hunk 3 above)
Comment (BHLoneWolf0) [reply docs/xf8bot/commands/info/memberinfo.md:25]: Oh i will add them

**Review by xf8b — APPROVED** [review]
looks good, going to merge #2 though

**Pull request closed without merging**